basics/datagrid/src/index.ts

```diff
@@ -2,17 +2,13 @@ import {
   JupyterFrontEnd,
   JupyterFrontEndPlugin
 } from '@jupyterlab/application';
-
-import { IMainMenu } from '@jupyterlab/mainmenu';
-
-import { Menu } from '@phosphor/widgets';
-
 import { ICommandPalette } from '@jupyterlab/apputils';
-
-import { StackedPanel } from '@phosphor/widgets';
+import { IMainMenu } from '@jupyterlab/mainmenu';
```

Comment:
Should we keep an empty line between imports?

Reply:
You are right, it helps readability (side effect of the automatic sort import algorithm).